### ISSUE TITLE
EstimateResponseSH dipy interface can not change default 'auto' input to False

### ISSUE THREAD

**Issue by salma1601** [nipy/nipype#1849]
### Summary
I can't find a way to set response estimation to recursive
### Actual behavior
Only auto_response can be used 
### How to replicate the behavior
```Python
import nipype.interfaces.dipy as ndp

estimate = ndp.EstimateResponseSH
estimate.inputs.auto = False
```
throughs the following error
>IOError: Input "recursive" is mutually exclusive with input "auto", which is already set

and similarly with
```Python
estimate.inputs.recursive = True
```

### Platform details:
{'nibabel_version': '2.1.0', 'networkx_version': '1.11', 'numpy_version': '1.11.2', 'sys_platform': 'linux2', 'sys_version': '2.7.6 (default, Oct 26 2016, 20:30:19) \n[GCC 4.8.4]', 'commit_source': 'installation', 'commit_hash': 'a4d1e07', 'pkg_path': '/home/bougacha/.local/lib/python2.7/site-packages/nipype', 'sys_executable': '/usr/bin/python', 'traits_version': '4.6.0', 'scipy_version': '0.18.1'}
0.12.1

**Comment by mgxd** [comment]
@salma1601 this should be fixed once #1856 is merged - feel free to use [my branch](https://github.com/mgxd/nipype/tree/fix/ersh) in the meantime

**Comment by salma1601** [comment]
Ok thanks. I think that the response type is not the same for recursive response and autoresponse.
